### DISCUSSION TITLE
Skyline Won't Show Private Contributions

### DISCUSSION THREAD

**Discussion by imaperson1060** [community/community#19111] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way for me to have Skyline show contributions to my private repository so I can print it? I tried to enable showing private contributions on my profile, but that didn't change anything. Do I need to wait for it to uncache?

**Comment by melvincwng** [comment, marked as answer]
@imaperson1060, please do the following:

1. Kindly check that the email used when committing is included in the email setting at GitHub. That's one of the more common reasons why the contributions aren't showing in the GitHub history.
2. For other possible reasons, please check out [this reference over here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile).
3. If option 1 & 2 don't work still, please raise a support ticket to [GitHub Support](https://support.github.com/). They will investigate the issue, and usually would get back to you within a few working days.

Hope this helps.

Thank you, and have a great day ahead 😃

**Reply by melvincwng** [reply]
@imaperson1060, if this answer has helped you, please kindly 'Mark as answer' on this reply so that others know that this issue has been addressed.

**Reply by imaperson1060** [reply]
The commits do appear on my profile, so I don't think it's the email. I do have 4 different emails on my account, but that shouldn't matter because all my public commits use the same email. I make my commits on GitHub Desktop so it shouldn't be a problem with that either. Thanks though!

**Reply by imaperson1060** [reply]
I just tried it again, and this time it showed everything, so it was just a caching problem (because I enabled private commits after going to Skyline).

**Comment by dev-velo** [comment]
I had the same issue and enabled the "Include private contributions on my profile" setting. In your Github profile settings, in the public profile section under "Contributions & Activity" make sure the checkbox for this setting is selected.
